core/clause_6_conventions.adoc

```diff
@@ -90,16 +90,30 @@ NOTE: OGC Web API standards may include a community-defined identifier as part o
 
 Additional information on this topic is provided in the link:http://docs.opengeospatial.org/DRAFTS/20-071.html[OGC API - Common Users Guide]. 
 
+[[link-conventions]]
+=== Links
+
+OGC Web API Standards use <<rfc8288,RFC 8288 (Web Linking)>>  to express relationships between resources. Resource representations defined in these standards commonly include an "links" element. A "links" element is an array of individual hyperlink elements. These "links" elements provide a convention for associating related resources.
+
+The individual hyperlink elements that make up a "links" element are defined using the following link:https://github.com/opengeospatial/oapi_common/blob/master/core/openapi/schemas/link.json[Hyperlink Schema].
+
+[source, JSON]
+include::openapi/schemas/link.json[]
+
+In addition, links should be passed in the response using HTTP link headers. These links are accessible to the client without a need to process the resource. 
+
+include::../general/recommendations/general/REC_link-header.adoc[]
+
 [[link-relation-conventions]]
 === Link relations
 
-<<rfc8288,RFC 8288 (Web Linking)>> is used to express relationships between resources. Link relation types from the https://www.iana.org/assignments/link-relations/link-relations.xhtml[IANA Link Relations Registry] are used wherever possible. Additional link relation types are registered with the https://github.com/opengeospatial/NamingAuthority/blob/master/incubation/linkRelationTypes/linkrelations.csv[OGC Link Relation Registry].
+Link relation types identify the semantics a link. For example, a link with the relation type "service-meta" indicates that the current link context has service metadata at the link target.
 
-The link relationships used in API-Common Core are described in <<link-relations-table>>.
+Link relation types are expressed using the "rel" property from the link:https://github.com/opengeospatial/oapi_common/blob/master/core/openapi/schemas/link.json[Hyperlink Schema]. 
 
-Each resource representation in this standard includes an array of links. Implementations are free to add additional links for all resources provided by the API. For example, an *enclosure* link could reference a bulk download of a collection. Or a *related* link on a feature could reference a related feature.
+The "rel" property is populated using values from the https://www.iana.org/assignments/link-relations/link-relations.xhtml[IANA Link Relations Registry] wherever possible. Additional values are registered with the https://github.com/opengeospatial/NamingAuthority/blob/master/incubation/linkRelationTypes/linkrelations.csv[OGC Link Relation Registry]. Additional relation type values can be used if neither of these registers suffice.
```

Comment:
OGC Link Relation Registry URL should be `https://github.com/opengeospatial/NamingAuthority/blob/master/registers/linkrelations.csv`

Reply:
Thanks @bradh. Both corrections have been made.